libs/lensfun/lens.cpp

```diff
@@ -11,7 +11,7 @@
 #include <regex>
 #include <string.h>
 #include <locale.h>
-#include <math.h>
+#include <cmath>
```

Comment:
Since this library extensively omits `std::` namespace, this change leads to unnecessary float to double conversion for all the mathematical functions. When I did it for `lfModifier` class, it made coordinates transformation ~10% slower.

Here is an example: https://godbolt.org/z/Ke1M53K96

But it's a matter for standard library and compiler. MSVS produces identical code, but clang and gcc treat mathematical functions without `std::` prefix from `cmath` header as double.

I'm sorry that such a little changes led to the bigger ones.

Reply:
Thanks for explaining the effect of the header change, didn't expect the difference in performance.
> I'm sorry that such a little changes led to the bigger ones.

Well, that's how it is.